tracing-agent/src/main/java/io/strimzi/tracing/agent/TracingAgent.java

```diff
@@ -4,34 +4,30 @@
  */
 package io.strimzi.tracing.agent;
 
-import io.jaegertracing.Configuration;
-import io.opentracing.Tracer;
-import io.opentracing.util.GlobalTracer;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-
 /**
- * A very simple Java agent which initializes the Jaeger Tracer
+ * A very simple Java agent which initializes the distributed tracing if requested
  */
 public class TracingAgent {
-    private static final Logger LOGGER = LoggerFactory.getLogger(TracingAgent.class);
 
     /**
      * Agent entry point
      *
      * @param agentArgs The type of tracing which should be initialized
      */
     public static void premain(String agentArgs) {
-        if ("jaeger".equals(agentArgs)) {
-            String jaegerServiceName = System.getenv("JAEGER_SERVICE_NAME");
-
-            if (jaegerServiceName != null) {
-                LOGGER.info("Initializing Jaeger tracing with service name {}", jaegerServiceName);
-                Tracer tracer = Configuration.fromEnv().getTracer();
-                GlobalTracer.registerIfAbsent(tracer);
-            } else {
-                LOGGER.error("Jaeger tracing cannot be initialized because JAEGER_SERVICE_NAME environment variable is not defined");
-            }
+        Tracing tracing;
+        switch (agentArgs) {
+            case "jaeger":
```

Comment:
I see that you are using "hardcoded" values on few places, would it make sense to use some constants, for example from the `api` module (like `TYPE_JAEGER`)?

Reply:
Tbh I wanted to avoid to add the `api` module dependency to the agent just for this ...

Reply:
Ah okay, make sense. Thanks :)